apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -228,6 +227,8 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
 
                 const paymentCategory = get(extra, 'paymentCategory', null)
 
+                let billingAccountFoundFlag = false
```

Comment:
It looks strange

Reply:
I changed the naming, so the code is more conventional now